### PR TITLE
Remove Cucumber Keywords from Sidebar Buttons

### LINKED ISSUE #5: Remove Cucumber Key-words from Sidebar Buttons
Currently the sidebar button text just has the full text of the feature/scenario title that includes the Cucumber keywords (Feature, Scenario, Scenario Outline).

It seems like these keywords are unnecessary since the button structure gives the user the context for what is being described. It also wastes valuable horizontal space in the sidebar which could be used to show more of the actual titles themselves.

### PR DESCRIPTION
**Pull request by jkuester**
Currently the sidebar buttons include Cucumber keywords ("Feature", "Scenario", and "Scenario Outline").  These do not provide a whole lot of value in the sidebar and they waste precious horizontal space.  These changes replace the keywords with single-character pointers that help break up the buttons without taking up so much space.

Example: 
[features.zip](https://github.com/cerner/cucumber-forge-report-generator/files/3437151/features.zip)

closes #5

### CODE REVIEW

File: features/support/usageSteps.js [hunk 1]
```diff
@@ -72,33 +70,34 @@ Then(/^the scenario buttons for the (first|second) feature (?:are|will be) expan
   const index = featureIndex === 'first' ? 0 : 1;
   const featureButtons = this.outputHTML.getElementsByClassName('feature-button');
   const selectedFeatureButton = featureButtons[index];
-  // eslint-disable-next-line no-unused-expressions
   expect(selectedFeatureButton.classList.contains('active')).to.be.true;
   const scenarioPanel = selectedFeatureButton.nextElementSibling;
-  // eslint-disable-next-line no-unused-expressions
   expect(scenarioPanel.style.maxHeight).to.not.be.empty;
```
Comment (TobiTenno) [inline features/support/usageSteps.js]: should we switch from expect to should? that might make it a bit clearer

File: src/Generator.js [hunk 2]
```diff
@@ -196,19 +196,26 @@ const populateHtmlIdentifiers = (features) => {
...
     featureButton.featureWrapperId = feature.featureWrapperId;
-    featureButton.title = feature.name;
+    featureButton.title = trimCucumberKeywords(feature.name, 'feature');
     featureButton.scenarioButtons = [];
     feature.scenarios.forEach((scenario) => {
       const scenarioButton = {};
       scenarioButton.id = scenario.scenarioButtonId;
       scenarioButton.scenarioId = scenario.scenarioId;
-      scenarioButton.title = scenario.name;
+      scenarioButton.title = `- ${trimCucumberKeywords(scenario.name, 'scenario', 'scenario_outline')}`;
```
Comment (TobiTenno) [inline src/Generator.js]: could we move that `-` to the actual report format so it's part of the template and not the string?

Comment (TobiTenno) [reply src/Generator.js]: (like down to line 26 on doc_template)

Comment (jkuester) [reply src/Generator.js]: Good call

File: features/support/usageSteps.js (hunk 1 above)
Comment (jkuester) [reply features/support/usageSteps.js]: Okay, so I have been reading about the Should notation out here: https://www.chaijs.com/guide/styles/. I am not familiar with Object.prototype, so maybe I am missing something, but how am I supposed to setup the should() function on the various objects I need to assert when I do not have references to those objects ahead of time (when I am requiring the should like they do in the example....)?

Comment (TobiTenno) [reply features/support/usageSteps.js]:
```js
const chai = require('chai');
chai.should();
```

That's all you need to do to set up should.

Comment (jkuester) [reply features/support/usageSteps.js]: Hmm, okay so I tried this and the tests ran.  But the `no-unused-expressions` linting errors remained.  Basically I did this:
```.js
const should = require('chai').should();
selectedFeatureWrapper.classList.contains('active').should.be.true;
```

The first line has the `no-unused-vars` warning and the second line has the `no-unused-expressions` warning.

Comment (TobiTenno) [reply features/support/usageSteps.js]: ah, i forgot about this in other projects. i typically apply a different set of lintings for chai...
[like this](https://github.com/WFCD/warframe-worldstate-data/blob/master/test/.eslintrc.json)

I forgot that we basically overrided a bit so it worked right

Comment (TobiTenno) [reply features/support/usageSteps.js]: so we can just keep using expect and not worry about the linting bits, basically.

Comment (jkuester) [reply features/support/usageSteps.js]: Done

**Pull request merged**